modules/ding_config_import/ding_config_import.module

```diff
@@ -0,0 +1,169 @@
+<?php
+
+/**
+ * @file
+ * Implementation of simple form to upload YAML file.
+ */
+
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Implements hook_menu().
+ */
+function ding_config_import_menu() {
+  $item = [];
+
+  $item['admin/config/ding/config'] = [
+    'title' => 'Upload configuration',
+    'description' => 'Upload and import configuration from YAML.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('ding_config_import_admin_form'),
+    'access arguments' => array('administer site configuration'),
```

Comment:
I wonder if `administer site configuration` is sufficiently strict for this?

The module effectively allows you to bypass access control for other configuration pages which may have their own access control settings which are more strict that `administer site configuration`.

We could also consider [marking the module as `hidden`](https://www.drupal.org/docs/7/creating-custom-modules/writing-module-info-files-drupal-7x#hidden) to avoid someone enabling it in a production environment.

Reply:
I see your point and will add hidden, but it may give issues on the non-platform.sh test sites.

Reply:
I see your point.

With the approach you will have to use Drush to enable the module. Testers will usually have to talk to a developer in these cases.

Another option would be to restrict access to user 1. Perhaps that is more feasible?